dotnet/src/SemanticKernel.Abstractions/SkillDefinition/IReadOnlySkillCollection.cs

```diff
@@ -48,8 +49,6 @@ public interface IReadOnlySkillCollection
     /// <summary>
     /// Get all registered functions details, minus the delegates
     /// </summary>
-    /// <param name="includeSemantic">Whether to include semantic functions in the list</param>
-    /// <param name="includeNative">Whether to include native functions in the list</param>
     /// <returns>An object containing all the functions details</returns>
-    FunctionsView GetFunctionsView(bool includeSemantic = true, bool includeNative = true);
+    IReadOnlyList<FunctionView> GetFunctionsView();
```

Comment:
This should be named `GetFunctionViews` as it returns a list of `FunctionView` instances. Current name reflects the old implementation.

Reply:
Done